python/ray/worker.py

```diff
@@ -427,6 +429,9 @@ def get_resource_ids():
         each pair consists of the ID of a resource and the fraction of that
         resource reserved for this worker.
     """
+    worker = global_worker
```

Comment:
global_worker.check_connected() ? `worker` is not used later.

Reply:
I used 

```
worker = global_worker
worker.check_connected()
```

because that's how it's done in the rest of this file. We should probably undo this at some point.